src/calendar/calendar.api.json

```diff
@@ -132,6 +132,11 @@
       "type": "boolean",
       "description": "Whether to hide the arrows"
     },
+    {
+      "name": "arrowsHitSlop",
+      "type": "null | Insets | number",
+      "description": "Left & Right arrows. Additional distance outside of the buttons in which a press is detected, default: 20"
```

Comment:
Please add `"default"` key with the value (20) instead of adding it to the description

Reply:
Added default, removed description